src/main/java/org/broadinstitute/consent/http/service/DarCollectionService.java

```diff
@@ -144,4 +155,30 @@ public DarCollection updateDarCollection(Integer collectionId, User user) {
     return collections;
   }
 
+  // fetch elections in bulk via dar reference ids
+  // if any election exists, throw a bad request
+  // else collect the dar reference ids where the status is NOT cancelled
+  // once collected, perform bulk update where status is updated with cancelled
```

Comment:
I keep questioning our own logic in making this choice about ANY ELECTION that ever existed, and then re-convince myself this is the right choice. Can we add some extra commentary here, anything that will help me not re-question myself again. Maybe something like:

> If an election has ever been created for any individual DAR in a collection, a researcher cannot 
> move the collection into canceled state. That task would need to be completed by an admin or
> DAC Chairperson.